src/styles/blocks/form-field.scss

```diff
@@ -0,0 +1,25 @@
+.form-field {
+  background-color: $bg-color;
+  box-sizing: border-box;
+  display: inline;
+  width: 100%;
+  border-bottom: 1px solid #fff;
+  padding: 16px;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 22px;
+  color: #fff;
+  margin-bottom: 40px;
```

Comment:
Please pay attention to your css properties order
https://9elements.com/css-rule-order/